### PR TITLE
Add scrolling background to onboarding

### PR DESCRIPTION
**Pull request by lucyxubroad**
## Overview

* Added scrolling background to the onboarding screens

## Changes Made

* Added backgrounds to Assets
* Added background to onboarding page view controller and made it scrolling as the user navigates to new page in onboarding.

## Test Coverage

* Tested on multiple phones and just visually inspected.

## Related PRs or Issues (delete if not applicable)

#24 

## Screenshots (delete if not applicable)

<details>

  <summary>iPhone 8e</summary>

ezgif com-video-to-gif-4

  
  

</details>

### CODE REVIEW

File: CoffeeChat/Controller/OnboardingPageViewController.swift [hunk 1]
```diff
@@ -17,15 +17,33 @@ protocol OnboardingPageDelegate: class {
...
+        for subview in view.subviews {
+            if let scrollView = subview as? UIScrollView {
+                scrollView.delegate = self
+            }
+        }
+
+        backgroundImage.image = UIImage(named: "onboardingBackground")
+        backgroundImage.contentMode =  .scaleAspectFill
+        view.insertSubview(backgroundImage, at: 0)
+        print(backgroundXPosition)
```
Comment (kevinchan159) [inline CoffeeChat/Controller/OnboardingPageViewController.swift]: remove `print`?

File: CoffeeChat/Controller/OnboardingPageViewController.swift [hunk 2]
```diff
@@ -17,15 +17,33 @@ protocol OnboardingPageDelegate: class {
...
+        backgroundImage.frame = CGRect(x: backgroundXPosition, y: 0, width: screenWidth, height: screenHeight)
```
Comment (kevinchan159) [inline CoffeeChat/Controller/OnboardingPageViewController.swift]: we can't use constraints?

File: CoffeeChat/Controller/OnboardingPageViewController.swift [hunk 3]
```diff
@@ -35,12 +53,18 @@ class OnboardingPageViewController: UIPageViewController {
...
         setViewControllers([onboardingPages[index]], direction: .forward, animated: true, completion: nil)
     }
 
     func backPage(index: Int) {
         setViewControllers([onboardingPages[index]], direction: .reverse, animated: true, completion: nil)
     }
+
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        // Update x position based on screen size when page scrolls
+        backgroundXPosition -= 0.04 * screenWidth
```
Comment (kevinchan159) [inline CoffeeChat/Controller/OnboardingPageViewController.swift]: wheres the `0.04` coming from?

Comment (lucyxubroad) [reply CoffeeChat/Controller/OnboardingPageViewController.swift]: I printed out how often scrollViewDidScroll gets called and it seems to be after the page has scrolled 3% - 4% of the screen so I just used that value cause I want the background to scroll the same amount as the page.

File: CoffeeChat/Controller/OnboardingPageViewController.swift (hunk 2 above)
Comment (lucyxubroad) [reply CoffeeChat/Controller/OnboardingPageViewController.swift]: hmm here, im setting the start of the background at a specific value and then I keep updating it in scrollViewDidScroll. is there a better way to do this?

File: CoffeeChat/Controller/OnboardingPageViewController.swift (hunk 1 above)
Comment (kevinchan159) [reply CoffeeChat/Controller/OnboardingPageViewController.swift]: bump ^

**Pull request merged**